iocore/net/quic/test/test_QUICPacketHeaderProtector.cc

```diff
@@ -187,23 +180,17 @@ TEST_CASE("QUICPacketHeaderProtector")
 
     REQUIRE(client->handshake(&msg3, &msg2_1) == 1);
     REQUIRE(client->handshake(&msg3, &msg2_2) == 1);
-#else
-    REQUIRE(client->handshake(&msg3, &msg2) == 1);
-#endif
+    REQUIRE(msg3);
 
     // NS
-    QUICHandshakeMsgs msg4;
-    uint8_t msg4_buf[MAX_HANDSHAKE_MSG_LEN] = {0};
-    msg4.buf                                = msg4_buf;
-    msg4.max_buf_len                        = MAX_HANDSHAKE_MSG_LEN;
-
-    REQUIRE(server->handshake(&msg4, &msg3) == 1);
-
-    QUICHandshakeMsgs msg5;
-    uint8_t msg5_buf[MAX_HANDSHAKE_MSG_LEN] = {0};
-    msg5.buf                                = msg5_buf;
-    msg5.max_buf_len                        = MAX_HANDSHAKE_MSG_LEN;
-    REQUIRE(client->handshake(&msg5, &msg4) == 1);
+    QUICHandshakeMsgs *msg4 = nullptr;
+    REQUIRE(server->handshake(&msg4, msg3) == 1);
+    REQUIRE(msg4);
+
+    QUICHandshakeMsgs *msg5 = nullptr;
+    REQUIRE(client->handshake(&msg5, msg4) == 1);
+    // Doesn't pass with BoringSSL (what's expected?)
+    // REQUIRE(msg5);
```

Comment:
Forgot to update this one.  It'll be `REQUIRE(msg5 == nullptr);` and I'll push the change on the next PR because my local branch has already other commits.

Reply:
Reenabled this test.